transformations/decontextualize_sentence/test.json

```diff
@@ -0,0 +1,46 @@
+{
+  "type": "Decontextualizer",
+  "test_cases": [
+    {
+      "class": "Decontextualize",
+      "inputs": {
+        "sentence": "More than a few CEOs say the red-carpet treatment tempts them to return to a heartland city for future meetings."
+      },
+      "outputs": [{
+        "sentence": "More than a few CEOs say the red - carpet treatment tempts them to return to a heartland city ."}
+      ]
```

Comment:
Okay, so there are two things here: 
While removing modifier arguments is perfectly okay to generate a grammatical sentence and that theoretically at least - (SRL) modifier arguments talk about the event - when, where, why, etc. So, directly removing them makes little sense as both sentences can't be interchangeable.
John drove the car to reach the hospital. --> John drove the car. (The 2nd one is not a paraphrase but is implied.)
But on the other hand, I do think this can be useful for tasks like entailment since the 2nd one is entailed by the first clearly. So, I would suggest you to modify the implementation a bit. You could extend the [SentencePairOperation](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/interfaces) rather than SentenceOperation and apply the modifier removal on the second sentence. In the list of tasks, you could add TaskType.Entailment
What do you think?

Reply:
Thanks @kaustubhdhole for your suggestion. Entailment seems to be the good target case for this transformation. 
I have updated the code and test cases accordingly.